recipes/quantumflow/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "quantumflow" %}
+{% set version = "1.2.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/quantumflow-{{ version }}.tar.gz
+  sha256: 72696f61f1af6376029ac290669b9b351739dc8d8ee0305abd6b0f8696e1ed9c
+  patches:
+    - 0001-Define-encoding.patch
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - setuptools-scm
+  run:
+    - decorator <5.0.0
+    - importlib-metadata
+    - matplotlib-base
+    - networkx
+    - numpy
+    - opt-einsum
+    - pillow
+    - python >=3.7
+    - scipy
+    - sympy >=1.6
+    - typing_extensions
+  run_constrained:
+    - cirq >=0.8.0
+    - qiskit >=0.24.0
+    - pyquil >=2.28.0
+
+test:
+  imports:
+    - examples
```

Comment:
If this is passing, that means it's installing the examples as an `examples` package/module in the global package namespace, which is not good.

Reply:
Yeah, I think `setup.cfg` needs something like:
```ini
package_dir =
    = quantumflow
```

Reply:
When installing with pip it doesn't show up though:
```zsh
Package            Version
------------------ -------
cycler             0.11.0
decorator          4.4.2
fonttools          4.28.2
importlib-metadata 4.8.2
kiwisolver         1.3.2
matplotlib         3.5.0
mpmath             1.2.1
networkx           2.6.3
numpy              1.21.4
opt-einsum         3.3.0
packaging          21.3
Pillow             8.4.0
pip                21.3.1
pyparsing          3.0.6
python-dateutil    2.8.2
quantumflow        1.2.0
scipy              1.7.3
setuptools         59.4.0
setuptools-scm     6.3.2
six                1.16.0
sympy              1.9
tomli              1.2.2
wheel              0.37.0
zipp               3.6.0
```

Reply:
Scratch that, it shows up in the packages folder. I'll put in a PR upstream but it seems like this is not necessarily conda-forge's problem. Or should we hold off with merging until then?

Reply:
Or *maybe* removing `examples/__init__.py`? I'm really not clear why that exists.

Reply:
I will open an issue and see what upstream thinks, let's see.

Reply:
Well, we don't want our users to end up with an `examples` folder in `site-packages`, so we should do *something* on the conda-forge side.

Reply:
Ok, le me try something - one sec.

Reply:
Unfortunately, the package is still installed and, to be honest, not completely sure why. If you have a second, would you mind having a look at my PR here as I'm slightly stuck: https://github.com/gecrooks/quantumflow-dev/pull/87 Thanks!

Reply:
Maybe the work-around specified in [this issue](https://github.com/pypa/setuptools/issues/2688) could work?

Reply:
Also doesn't work. Hmm, this is really strange. I'll ask upstream.

Reply:
New release fixed it: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=424116&view=logs&jobId=6f142865-96c3-535c-b7ea-873d86b887bd&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823